pages/network/[network]/loans/[id].tsx

```diff
@@ -142,9 +142,8 @@ function LoansInner({
         kind: 'success',
         message: (
           <p>
-            {`You've successfully created loan #${loan.id}! To get notifications on its activity, go to the`}
+            {`You've successfully created loan #${loan.id}! To get notifications on its activity, go to the `}
```

Comment:
this is just here because we got a runtime error because `<Link>` is only supposed to have one child and the extra space counted as a distinct entity :man_shrugging: